### DISCUSSION TITLE
y291ga - ONVIF Profile S or T - Videoloft

### DISCUSSION THREAD

**Discussion by z33k666** [roleoroleo/yi-hack-Allwinner-v2#663] [Q&A]
I am trying to connect some Yi Home 1080P cameras, y291ga - 0.2.7, to our company's Videoloft appliance.

Videoloft says they support "ONVIF profiles S and T".

Which ONVIF Profiles are available in Yi-Hack?

Link to Videoloft ONVIF Guide for reference:
https://assetcdn.manything.com/portal-materials/app.videoloft.com/Install%20guides/Installer/Videoloft-InstallGuide-ONVIF.pdf

**Comment by roleoroleo** [comment]
Profile S.
